tests/lightning_tensor/test_lightning_tensor.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the generic lightning tensor class.
+"""
+
+
+import numpy as np
+import pennylane as qml
+import pytest
+from conftest import LightningDevice  # tested device
+from pennylane.wires import Wires
+
+from pennylane_lightning.lightning_tensor import LightningTensor
+
+if not LightningDevice._new_API:
+    pytest.skip("Exclusive tests for new API. Skipping.", allow_module_level=True)
```

Comment:
```suggestion
from conftest import LightningDevice, device_name  # tested device
from pennylane.wires import Wires

from pennylane_lightning.lightning_tensor import LightningTensor

if device_name != "lightning.tensor:
    pytest.skip("Exclusive tests for LightningTensor. Skipping.", allow_module_level=True)
```

Reply:
Second that, better use `LightningDevice, device_name` everywhere right away.